src/Sentry.AspNetCore/SentryAspNetCoreLoggerProvider.cs

```diff
@@ -1,14 +1,15 @@
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Options;
 using Sentry.Extensions.Logging;
+using Sentry.Infrastructure;
 
 namespace Sentry.AspNetCore;
 
 /// <summary>
-/// Logger provider for Sentry.
+/// Sentry Logger Provider for <see cref="Breadcrumb"/> and <see cref="SentryEvent"/>.
 /// </summary>
 [ProviderAlias("Sentry")]
-public class SentryAspNetCoreLoggerProvider : SentryLoggerProvider
+internal sealed class SentryAspNetCoreLoggerProvider : SentryLoggerProvider
```

Comment:
note: I made all `ILoggerProvider`-implementations (and derived) `internal`, so that these cannot be used outside of `AddSentry`/`UseSentry` where the Logger-Filter-Rules may not be correctly setup when adding manually

also, we were a bit inconsistent, as the derived types in MAUI were already `internal`